### ISSUE TITLE
ODBC no longer sees MS-Access 2013 views (queries) in QGIS v3.10.2 compared with v.3.10.0 where tables & views worked

### ISSUE THREAD

**Issue by MarcDucharme62** [qgis/QGIS#34463]
**Describe the bug**

**How to Reproduce**

**QGIS and OS versions**

**Additional context**

**Comment by MarcDucharme62** [comment]
ODBC Issue in QGIS 3.10.2: ODBC now only sees MS-Access 2013 tables but no longer sees the  views (queried tables) which was fully functional in QGIS 3.10.0 and now requires a password to proceed.

**Comment by MarcDucharme62** [comment]
ODBC no longer sees MS-Access 2013 views (queries) in QGIS v3.10.2 compared with QGIS v.3.10.0 where both tables & views were available

**Comment by MarcDucharme62** [comment]
Apologies for closing... ODBC Issue not not yet resolved.

**Comment by gioman** [comment]
@MarcDucharme62 I'm not familiarized with this format so I would appreciate of you can add the necessary steps to add the supoort in QGIS and as well as sample dataset, with expected results vs obtained.

**Comment by MarcDucharme62** [comment]
Thanks for your reply. I'm not sure what steps I'm expected to provide?
What I can say is that the ODBC links to a MS-Access database views
(queried tables) I had set-up successfully in the recent QGIS 3.10.0 are no
longer available since I've upgraded to QGIS 3.10.2. I'm using the MS
Office Plus 2013 Access. With the recent upgrade, now only the tables are
available but not the views (queries) of combined relational tables needed
to link in QGIS. Thanks.

Marc

**Comment by gioman** [comment]
> Thanks for your reply. I'm not sure what steps I'm expected to provide?

@MarcDucharme62 I use just more advanced data sources (PostGIS, GPKGs, etc.) and I have no idea how I would open an Access datasource in QGIS (I also do not use or own any copy of MS Office), so the request to add the detailed steps to 1) add the necessary drivers 2) open the data Also (again) please add a sample dataset, one that can be used to see the issue.

**Comment by MarcDucharme62** [comment]
I was successful in resolving my ODBC connection issue in QGIS previously posted. After testing the ODBC connection to a newly created MS-Access database (tables and views) with success, I concluded that the initial connection problem I encountered was due to an incorrect ODBC parameters path to a MS-Access dataset not containing the expected queried tables (views). Thanks for the quick response.

Best regards,

Marc
